flank-bash/scripts/android.ops.main.kts

```diff
@@ -0,0 +1,106 @@
+@file:Repository("https://dl.bintray.com/jakubriegel/kotlin-shell")
```

Comment:
should we run detekt across the kts files as they are kotlin? can we ?
WDYT?

Reply:
This could be complicated, cause scripts are standalone without `build.gradle.kts` file and detekt configuration

Reply:
We decided, to make scripts be part of flankScripts, so detekt is available